.gitignore

```diff
@@ -159,3 +159,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# prettier ignore
+.prettierignore
```

Comment:
Just in case you're wondering, Prettier is an extension I have to fix my formatting locally, but it kept breaking the app. I had to add a prettierignore file to stop it running on certain files, but then I added it to the gitignore so that it wouldn't be added to the repo.